airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/destination/StandardNameTransformer.java

```diff
@@ -41,7 +41,7 @@ public String getRawTableName(String streamName) {
 
   @Override
   public String getTmpTableName(String streamName) {
-    return convertStreamName("_airbyte_" + Instant.now().toEpochMilli() + "_" + streamName);
+    return convertStreamName("_airbyte_" + Instant.now().toEpochMilli() + "_" + getRawTableName(streamName));
```

Comment:
Hey @masonwheeler,

I am curious to know if there was any particular reason for this change?

Instead of `_airbyte_20120010310_streamname` as TmpTableName, it is now `_airbyte_20120010310__airbyte_raw_streamname` which makes the temporary names even longer than before...

Reply:
@ChristopheDuong I don't think that's true. He did a lift and shift of the original behavior as far as I can tell. Here's a screenshot of the logic he replaced.

![Screen Shot 2021-06-15 at 2 47 36 PM](https://user-images.githubusercontent.com/9092207/122127836-a0dee100-cde8-11eb-8f7b-7abd1454de31.png)

I agree the current naming is needlessly verbose, but that didn't happen as part of this change. The logic just moved.

The reason we pushed this logic into here is to avoid hacks like this: https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/JdbcBufferedConsumerFactory.java#L114-L120. Essentially by pushing the logic into the name transformer it is not configurable specifically for each database based on their specific needs (you'll see that the oracle name transformer handles shortening names).

By all means though, we can change the standard naming convention to something less silly.